### PR TITLE
Issue 52 missing field during filter

### PR DESCRIPTION
**Pull request by num8er**
Based on Discussion #52  I wrote test to check if it's a bug or not.

Currently I don't see such issue.
I switched `silent` to `false` and `true` same expected result.

image

### CODE REVIEW

**Comment by num8er** [comment]
@abbasudo  can we merge this PR as example of multiple fields in relation?

**Comment by num8er** [comment]
@IndexZer0 please check this branch and compare to Your code

**Review by abbasudo — APPROVED** [review]
@num8er examples should be listed in the documentation. However, tests are always welcome.

File: tests/Feature/FilterableByMultipleFieldInRelationTest.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                    ],
+                ],
+            ],
+        ];
+
+        $query = Author::with(['books'])
+            ->filter($filters);
+        $results = $query->get();
+
+//        echo "\n#######################";
```
Comment (abbasudo) [inline tests/Feature/FilterableByMultipleFieldInRelationTest.php:103]: remove these comments please

**Review by abbasudo — CHANGES_REQUESTED** [review]
Great job

File: tests/Feature/FilterableByMultipleFieldInRelationTest.php (hunk 1 above)
Comment (num8er) [reply tests/Feature/FilterableByMultipleFieldInRelationTest.php:103]: Asap during the day.
Thanks

Comment (num8er) [reply tests/Feature/FilterableByMultipleFieldInRelationTest.php]: @abbasudo  done

**Review by abbasudo — APPROVED** [review]

**Pull request merged**